operations/research-and-development/product/pm-onboarding.md

```diff
@@ -0,0 +1,240 @@
+  * Learn about their areas of ownership
+* Participate in PM, R&D, and Customer Obsession Meetings
+* Jira
+  * Sign up to Jira and post in the “Jira configuration” channel to be added to internal team
+  * Get Jira training from PM buddy
+    * [Mattermost Platform](https://mattermost.atlassian.net/projects/MM/issues/MM-10977?filter=allopenissues)
+* Make your first edit and pull request on GitHub
+  * Update documentation or product help text
+* Small contributions
+  * File a bug in Jira
+  * Test a new feature on a test server and suggest an improvement
+  * Leave feedback on a design proposal
+
+Readings:
+* The Advantage:
+  * [https://drive.google.com/file/d/1zOV4nqVPxRHsQrxcqQwkGq2hCvB2CKJU/view](https://drive.google.com/file/d/1zOV4nqVPxRHsQrxcqQwkGq2hCvB2CKJU/view)
+* The Five Dysfunctions of a Team: A Leadership Fable
+  * [https://drive.google.com/file/d/1yvLMW5tHGoTTjrq5VY17SCoeQoNXn9jA/view?usp=sharing](https://drive.google.com/file/d/1yvLMW5tHGoTTjrq5VY17SCoeQoNXn9jA/view?usp=sharing)
+
+## First 30 days
+
+### Objective: Build foundational knowledge
+
+### Training focus:
+
+* Users and customers
+* Product
+  * Features/Functions
+  * Technical
+* Market and Competitive Landscape
+* Team
+  * People and Processes
+
+### Activities:
+Learn about our customers and users
+* Support
+  * Customers: Zendesk \(Sven, Support Lead)\)
+    * Meet with Sven for Zendesk training and support overview
+    * Join support standups for a week or two, and help out on support tickets
+  * Community: GitHub, Forums \(Amy\)
+* Customer Success \(Meet with Jenn\)
+  * Overview of Customer Success team, who owns what
+  * Current Customer Success processes
+  * Typical challenges during customer onboarding
+  * Join one CSM team meeting to meet the rest of the CS team
+* Customer calls
+  * Join calls with CSMs
+  * Join calls with Sales team
+* Feature requests
+  * [Uservoice forums](https://mattermost.uservoice.com/)
+  * [Process feedback in productboard](https://mattermost.productboard.com/insights/shared-inbox) 
+* Learn about the market
+* Sales \(Regional VPs and Customer Engineers\)
+  * Sales team overview
+  * Sales team priorities
+  * What competitors are we up against
+  * How does Sales team position the product
+  * What are typical challenges for the Sales team
+  * Which sales do we win, which do we lose
+  * What are the most common questions on configurations and trials
+* Marketing \(Marketing Team Lead\)
+  * Marketing team overview
+  * What are the Marketing team’s priorities
+  * What is our current messaging and positioning around Mattermost
+  * What is needed from Product Managers
+* Mini competitive research projects
+    * Review of MS Teams, Slack, Discord, and other chat products
+* Spend a week doing customer support
+  * Review documentation first and work with Support Lead or PM mentor to answer any questions documentation does not address
+  * Update docs with answers as appropriate
```

Comment:
I'd like to see Documentation added to this list as a PM will work with a writer and I've found that an intro meeting really helps. It's also really beneficial to the writers to get fresh-eye feedback! :)

Reply:
Added to the first week as part of the R&D team introductions.